modules/apps/frontend-taglib/frontend-taglib/src/main/java/com/liferay/frontend/taglib/servlet/taglib/ScreenNavigationRegistryUtil.java

```diff
@@ -16,19 +16,18 @@
 import java.util.Collections;
 import java.util.List;
 
+import org.osgi.framework.Bundle;
 import org.osgi.framework.BundleContext;
-import org.osgi.service.component.annotations.Activate;
-import org.osgi.service.component.annotations.Component;
-import org.osgi.service.component.annotations.Deactivate;
+import org.osgi.framework.FrameworkUtil;
 
 /**
  * @author Eudaldo Alonso
  */
-@Component(service = ScreenNavigationRegistry.class)
-public class ScreenNavigationRegistry {
+public class ScreenNavigationRegistryUtil {
```

Comment:
Hey @GabrielHenriquedev. please fix the commit message so it matches your changes, thanks.